### PR TITLE
- fix Photos not shown in large browser windows #630

### LINKED ISSUE #630: Photos not shown in large browser windows
**Describe the bug**
Photo gallery doesn't work when browser window is large.

**To Reproduce**
1. Open a maximized browser window on a 4k display
2. Click on the Photos gallery
3. Observe white background with a rotating waiting indicator. The photos are not displayed.
4. This issue doesn't occur in other (default) NC applications or when the window size is smaller (~1920x1080 already works fine).

**Desktop (please complete the following information):**
 - OS: Linux, Ubuntu 20.04
 - Browser: Tested on Firefox and Chromium

### PR DESCRIPTION
**Pull request by dsmic**
As the title says: this should fix this issue, as the max: key was not handled properly before

Fix #630

### CODE REVIEW

**Comment by dsmic** [comment]
Sorry, I don't know how to fix the last missing check, but the code changes are minimal, so you could probably merge it without those changes....

File: src/services/GridConfig.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ export default new Vue({
 	methods: {
 		handleWindowResize() {
 			// find the first grid size that fit the current window width
-			const currentSize = Object.keys(sizes).find(size => size > document.documentElement.clientWidth)
+			const currentSize = Object.keys(sizes).find(size => size > document.documentElement.clientWidth || size === 'max')
 			this.gridConfig = sizes[currentSize]
```
Comment (skjnldsv) [inline src/services/GridConfig.js:48]: I would prefer the fallback on the this.gridConfig assignment instead.
```suggestion
			// fallback to max if none matches
			this.gridConfig = sizes[currentSize] || sizes['max']
```

Comment (dsmic) [reply src/services/GridConfig.js:48]: This seems to be more elegant, sorry I am not a javascript guy 😊😊

Comment (skjnldsv) [reply src/services/GridConfig.js:48]: No worries :)

Can you do the change so I can approve? :)

**Comment by dsmic** [comment]
OK, still fighting with the automatic checks. 
With your suggestion I got the Error ['max'] better writen in dot notation, so I did it :)

Thanks for teaching the power of || in javascript, Detlef

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Thanks @dsmic 
Next time you open a pr, could you enable the owners of the repo to push to your fork? That help us rebase things without having to wait for you if necessary :)

Because I can't rebase now, please either enable write permissions or rebase yourself this branch to master :wink:

**Comment by skjnldsv** [comment]
> Because I can't rebase now, please either enable write permissions or rebase yourself this branch to master

Ah, my bad, it seems we have the permissions, no idea why my push failed.

**Comment by skjnldsv** [comment]
Yeah, I can rebase, @dsmic please rebase, and drop changes on /js so we can update the bundles again :)

**Comment by dsmic** [comment]
Sorry, but I don't know, why it closed the pull request. I cleaned my fork ...
In case you still have trouble I added you to the fork...

**Comment by skjnldsv** [comment]
> Sorry, but I don't know, why it closed the pull request. I cleaned my fork ...

Ah, you didn't rebased to latest master!
We need to ship the compiled files, but if you're not up-to-date, there will be conflicts.

**Comment by dsmic** [comment]
Am I supposed to commit my changes from the build?

> # Zum Commit vorgemerkte Änderungen:
#       geändert:       js/photos-0.js
#       geändert:       js/photos-0.js.map
#       geändert:       js/photos-1.js
#       geändert:       js/photos-1.js.map
#       geändert:       js/photos-10.js
#       geändert:       js/photos-10.js.map
#       geändert:       js/photos-2.js
#       geändert:       js/photos-2.js.map
#       geändert:       js/photos-3.js
#       geändert:       js/photos-4.js
#       geändert:       js/photos-4.js.map
#       geändert:       js/photos-5.js
#       geändert:       js/photos-5.js.map
#       geändert:       js/photos-6.js
#       geändert:       js/photos-6.js.map
#       geändert:       js/photos-7.js
#       geändert:       js/photos-7.js.map
#       geändert:       js/photos-8.js
#       geändert:       js/photos-8.js.map
#       geändert:       js/photos-9.js
#       geändert:       js/photos-9.js.map
#       geändert:       js/photos-main.js
#       geändert:       js/photos-main.js.map
#       geändert:       js/photos-service-worker.js
#
# Unversionierte Dateien:
#       js/photos-3.js.map
#

**Comment by skjnldsv** [comment]
> Am I supposed to commit my changes from the build?

Yes, this repo expect npm bundles to be up--to-date before merging

**Comment by dsmic** [comment]
I still get node12.x failing:

`Uncommited changes in webpack build
HEAD detached at pull/664/merge
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   js/photos-0.js
	modified:   js/photos-0.js.map
	modified:   js/photos-1.js
	modified:   js/photos-1.js.map
	modified:   js/photos-10.js
	modified:   js/photos-10.js.map
	modified:   js/photos-2.js
	modified:   js/photos-2.js.map
	modified:   js/photos-4.js
	modified:   js/photos-4.js.map
	modified:   js/photos-5.js
	modified:   js/photos-5.js.map
	modified:   js/photos-6.js
	modified:   js/photos-6.js.map
	modified:   js/photos-7.js
	modified:   js/photos-7.js.map
	modified:   js/photos-8.js
	modified:   js/photos-8.js.map
	modified:   js/photos-9.js
	modified:   js/photos-9.js.map
	modified:   js/photos-main.js
	modified:   js/photos-main.js.map
	modified:   js/photos-service-worker.js

no changes added to commit (use "git add" and/or "git commit -a")
Error: Process completed with exit code 1.`

**Pull request closed without merging**